.github/actions/nss/action.yml

```diff
@@ -86,21 +86,21 @@ runs:
         fi
 
     - name: Checkout NSS
-      if: env.BUILD_NSS == '1'
+      if: env.USE_SYSTEM == '0'
```

Comment:
USE_SYSTEM_NSS !

Reply:
Uff, good catch. Thank you @martinthomson! Will be fixed in https://github.com/mozilla/neqo/pull/2145.